### ISSUE TITLE
Option to disable updates entirely

### ISSUE THREAD

**Issue by sippeangelo** [lloesche/valheim-server-docker#618]
Valheim is updating almost daily, feels like, and it's really hard to keep up with keeping mods and the alternative branch of ValheimPlus working. We just want to stay on an old version but the container keeps updating on start even though I set UPDATE_CRON to blank.

**Comment by Shawak** [comment]
Blank probably sets it back to a defualt value, you could try a specfic date far in the future.

**Comment by makarov-roman** [comment]
must have for modded server

**Comment by Beermaster90** [comment]
I agree completely way to disable updates in docker startup / for mods would be nice. Forcing game to stay in stable version with mods.

**Comment by kiloutyg** [comment]
Is there any solution to stop update at starting through the envvars? Or should we fork the repo and propose a solution? Is it even possible to specify a version to limit repercussion on mods ?
